### PR TITLE
Adding /Debug:FastLink support for VS2015.

### PR DESCRIPTION
**Pull request by tvandijck**
and explicitly handle `symbols 'Off'`

### CODE REVIEW

File: tests/actions/vstudio/vc2010/test_rule_vars.lua [hunk 1]
```diff
@@ -100,11 +100,11 @@
 
 	function suite.onPathVar()
 		createVar { name="MyVar", kind="path" }
-		myRuleVars { MyVar = "../../path/to/file" }
+		myRuleVars { MyVar = "../path/to/file" }
```
Comment (tvandijck) [inline tests/actions/vstudio/vc2010/test_rule_vars.lua]: this is actually a separate bugfix that I hit.. if you go up to far, the path becomes absolute, and stays absolute. so ../.. actually goes up beyond the premake folder itself, and hence in our case where it is in "c:\dev\premake", the patch would actually become "C:\path\to\file", which then subsequently stays absolute.

File: tests/actions/vstudio/vc2010/test_excluded_configs.lua [hunk 2]
```diff
@@ -51,7 +51,6 @@
 		test.capture [[
 <Link>
 	<SubSystem>Console</SubSystem>
-	<GenerateDebugInformation>false</GenerateDebugInformation>
```
Comment (tvandijck) [inline tests/actions/vstudio/vc2010/test_excluded_configs.lua]: Default behavior is to not output this flag at all..

File: src/actions/vstudio/vs2010_vcxproj.lua [hunk 3]
```diff
@@ -1243,10 +1243,12 @@
 			else
 				value = "EditAndContinue"
 			end
+		elseif cfg.symbols == p.OFF then
+			value = ""
```
Comment (TurkeyMan) [inline src/actions/vstudio/vs2010_vcxproj.lua]: This isn't right; `symbols 'off'` will not emit any XML. In VS, the default for symbols is `ProgramDatabase`...
It should be `value = "None"`, and if `symbols` is never called, only then should the element not be written.

Comment (TurkeyMan) [inline src/actions/vstudio/vs2010_vcxproj.lua]: #541 fixes `symbols` it more properly. This PR should be split so the different fixes can be merged independently.

Comment (TurkeyMan) [inline src/actions/vstudio/vs2010_vcxproj.lua]: And #544 implements the symbols api properly...

**Comment by tvandijck** [comment]
Rebased and updated to just do the FastLink option.

**Comment by samsinsane** [comment]
We should have some tests for this new output, don't want it to get broken. Also, the VS2015 license seems to have expired for AppVeyor? Did you have to provide a license?

**Comment by tvandijck** [comment]
> Also, the VS2015 license seems to have expired for AppVeyor? Did you have to provide a license?

I contacted AppVeyor already about that... this is on their end to fix unfortunately...

**Pull request merged**